common/settings.h

```diff
@@ -136,6 +136,7 @@ struct Settings {
   std::optional<std::vector<std::string>> trace_skia_allowlist;
   bool trace_startup = false;
   bool trace_systrace = false;
+  bool disable_timeline_event_handler = false;
```

Comment:
Consider making this `enable_timeline_event_handler` and defauling it to true (https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-double-negatives-in-apis)

Reply:
done